wam: use webos_ports_fork_repo with default branch

shr-project opened the pull request:
git log --oneline d817c5a083127fc6e2ffb036ae3b4912c43c0262..823860dbb16fdb91e503a194d6dab10f8752dff0
823860d (HEAD -> webOS-ports/master, gg/webOS-ports/master,
herrie/LuneOS-rebased-honister) WebAppBase: add noWindow support
51c4861 Add window.open support
dc1518e Add relaunch support for legacy enyo apps
d73cbf1 DeviceInfoImpl: add "tvDeviceInfo" data
3082dd5 Add basic implementation for getResource for webOSSystem
2bffb21 Fix build with gcc-11

0002-add_oomd_option.patch was added in:
  commit a5e2d22994a8f781c70ea1fbe6986cb354dace70
  Author: Herman van Hazendonk <github.com@herrie.org>
  Date:   Wed Feb 9 08:46:06 2022 +0100

    Updates for WAM

but never included in SRC_URI.

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

Herrie82 commented:
@shr-project No objections in general, I guess we might need to pull in a few more upstream commits too anyway. I guess this somehow got missed with all the migrations going on at the same time

shr-project commented:
> @shr-project No objections in general, I guess we might need to pull in a few more upstream commits too anyway. I guess this somehow got missed with all the migrations going on at the same time

I've changed the branch name to start with OSE submission number so that we can more easily rebase our patches on top of newer submission without force pushing to webOS-ports/master branch.

I've also added the fix for DEBUG_BUILD, thanks @Tofee!.
